<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogfileEditor</class>
 <widget class="QMainWindow" name="LogfileEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1502</width>
    <height>774</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Logfile editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../assets/resource.qrc">
    <normaloff>:/linescaleGUI/logo/logo.png</normaloff>:/linescaleGUI/logo/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QSplitter" name="splitterUpperHorizontal">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="childrenCollapsible">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="widgetFileBrowser" native="true">
        <layout class="QVBoxLayout" name="groupFileBrowser">
         <item>
          <widget class="QTreeWidget" name="treeLogfiles">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>0</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="indentation">
            <number>0</number>
           </property>
           <property name="uniformRowHeights">
            <bool>false</bool>
           </property>
           <property name="itemsExpandable">
            <bool>true</bool>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <property name="headerHidden">
            <bool>false</bool>
           </property>
           <column>
            <property name="text">
             <string>Filename</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>DeviceID</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Date</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Time</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>LogNr</string>
            </property>
           </column>
           <item>
            <property name="text">
             <string>e34r34.csv</string>
            </property>
            <property name="text">
             <string>34:34:45</string>
            </property>
            <property name="text">
             <string>15.05.23</string>
            </property>
            <property name="text">
             <string>23:23:34</string>
            </property>
            <property name="text">
             <string>001</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>e34r33444.csv</string>
            </property>
            <property name="text">
             <string>34:34:45</string>
            </property>
            <property name="text">
             <string>15.05.23</string>
            </property>
            <property name="text">
             <string>23:33:34</string>
            </property>
            <property name="text">
             <string>002</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>e34r334.csv</string>
            </property>
            <property name="text">
             <string>34:34:48</string>
            </property>
            <property name="text">
             <string>15.05.23</string>
            </property>
            <property name="text">
             <string>00:23:34</string>
            </property>
            <property name="text">
             <string>003</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>e34r234.csv</string>
            </property>
            <property name="text">
             <string>34:34:46</string>
            </property>
            <property name="text">
             <string>15.05.23</string>
            </property>
            <property name="text">
             <string>23:25:34</string>
            </property>
            <property name="text">
             <string>004</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>e342r34.csv</string>
            </property>
            <property name="text">
             <string>34:34:47</string>
            </property>
            <property name="text">
             <string>15.05.23</string>
            </property>
            <property name="text">
             <string>23:43:34</string>
            </property>
            <property name="text">
             <string>010</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="layoutButtonsFile">
           <item>
            <widget class="QPushButton" name="btnRemoveFile">
             <property name="text">
              <string>Remove selected file</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnLoadFile">
             <property name="text">
              <string>Load new file</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupSelLogfile">
        <property name="title">
         <string>Selected logfile</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="metaData">
           <item>
            <layout class="QFormLayout" name="leftMetaData">
             <item row="0" column="0">
              <widget class="QLabel" name="lblDeviceID">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>DeviceID</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="outDeviceID">
               <property name="text">
                <string>-</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="textInteractionFlags">
                <set>Qt::TextSelectableByMouse</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="lblDate">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Date</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="outDate">
               <property name="text">
                <string>-</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="textInteractionFlags">
                <set>Qt::TextSelectableByMouse</set>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="lblTime">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Time</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="outTime">
               <property name="text">
                <string>-</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="textInteractionFlags">
                <set>Qt::TextSelectableByMouse</set>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="lblLogNr">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>LogNo</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="outLogNr">
               <property name="text">
                <string>-</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="textInteractionFlags">
                <set>Qt::TextSelectableByMouse</set>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="lblUnit">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Unit</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLabel" name="outUnit">
               <property name="text">
                <string>-</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="textInteractionFlags">
                <set>Qt::TextSelectableByMouse</set>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="lblRelZero">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>RelZero</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLabel" name="outRelZero">
               <property name="text">
                <string>-</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="textInteractionFlags">
                <set>Qt::TextSelectableByMouse</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QFormLayout" name="middleMetaData">
             <item row="0" column="0">
              <widget class="QLabel" name="lblSpeed">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Speed</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="outSpeed">
               <property name="text">
                <string>-</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="textInteractionFlags">
                <set>Qt::TextSelectableByMouse</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="lblTriggerForce">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Trigger force</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="outTriggerForce">
               <property name="text">
                <string>-</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="textInteractionFlags">
                <set>Qt::TextSelectableByMouse</set>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="lblStopForce">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Stop force</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="outStopForce">
               <property name="text">
                <string>-</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="textInteractionFlags">
                <set>Qt::TextSelectableByMouse</set>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="lblPreCatch">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Pre catch</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="outPreCatch">
               <property name="text">
                <string>-</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="textInteractionFlags">
                <set>Qt::TextSelectableByMouse</set>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="lblCatchTime">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Catch time</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLabel" name="outCatchTime">
               <property name="text">
                <string>-</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="textInteractionFlags">
                <set>Qt::TextSelectableByMouse</set>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="lblTotalTime">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Total time</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLabel" name="outTotalTime">
               <property name="text">
                <string>-</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="textInteractionFlags">
                <set>Qt::TextSelectableByMouse</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QFormLayout" name="rightMetaData">
             <item row="0" column="0">
              <widget class="QLabel" name="lblMinForce">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Minimum force</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="outMinForce">
               <property name="text">
                <string>-</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="textInteractionFlags">
                <set>Qt::TextSelectableByMouse</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="lblMinTime">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Timestamp Fmin</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="outMinTime">
               <property name="text">
                <string>-</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="textInteractionFlags">
                <set>Qt::TextSelectableByMouse</set>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="lblMaxForce">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Maximum force</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="outMaxForce">
               <property name="text">
                <string>-</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="textInteractionFlags">
                <set>Qt::TextSelectableByMouse</set>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="lblMaxTime">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Timestamp Fmax</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="outMaxTime">
               <property name="text">
                <string>-</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="textInteractionFlags">
                <set>Qt::TextSelectableByMouse</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="lineSelLogfile">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="layoutPlotSelected"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="layoutPlotMerged"/>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layoutBottom">
      <item>
       <widget class="QLabel" name="outLogLine">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="text">
         <string>Successfully loaded e342r34.csv from file system</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnExportFile">
        <property name="text">
         <string>Export file</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1502</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit_file_editor"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAdd_logfile_from_disk"/>
    <addaction name="actionRemove_all_files"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionOpen_online_help"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionFileBrowser"/>
    <addaction name="actionSelectedLogfile"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionExit_file_editor">
   <property name="text">
    <string>Exit file editor</string>
   </property>
  </action>
  <action name="actionAdd_logfile_from_disk">
   <property name="text">
    <string>Load new file</string>
   </property>
  </action>
  <action name="actionRemove_all_files">
   <property name="text">
    <string>Remove all files</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionOpen_online_help">
   <property name="text">
    <string>Open online help</string>
   </property>
  </action>
  <action name="actionFileBrowser">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>File browser</string>
   </property>
  </action>
  <action name="actionSelectedLogfile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Selected logfile</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../assets/resource.qrc"/>
 </resources>
 <connections/>
</ui>
